src/styles/app.css

```diff
@@ -0,0 +1,18 @@
+body{
+  background-color: rgba(0, 0, 0, 0.541);
+  color: white;
+  font-family: 'Courier New', Courier, monospace;
+}
+
+h1{
+  display: flex;
+  justify-content: center;
+}
+.center{
+  display: flex;
+  justify-content: center;
+}
+
+
+
+  
```

Comment:
overall colour scheme isn't the most inspiring, I get the github dark mode inspiration but the grey that you used isn't my favourite (but I'm just being picky)